src/_locales/en_US/messages.json

```diff
@@ -43,6 +43,10 @@
         "message": "Help us by donating and sharing your support for our tools",
         "description": "Shown below the Donate button on the intro page."
     },
+    "intro_link_policy": {
+        "message": "<a href='https://www.eff.org/code/privacy/policy' target='_blank'>Privacy Policy</a>",
```

Comment:
We should move the anchor link out of the translation message into the HTML document. No need to complicate translations in this case, I think.